docs/products/mysql/howto/migrate-database-mysqldump.rst

```diff
@@ -0,0 +1,122 @@
+Backup and restore MySQL data using ``mysqldump``
+=================================================
+
+Performing the backup of your MySQL data to another storage service is a good way to ensure access to your data in case a failure occurs. In this article, you can find out how to copy your Aiven for MySQL data to a file and backup to another Aiven for MySQL database using ``mysqldump`` `tool <https://dev.mysql.com/doc/refman/8.0/en/mysqldump.html>`__.
+
+Prerequisites
+-------------
+
+* The ``mysqldump`` `tool <https://dev.mysql.com/doc/refman/8.0/en/mysqldump.html>`_ installed. Check out the `official MySQL <https://dev.mysql.com/doc/mysql-shell/8.0/en/mysql-shell-install.html>`_ documentation on how to install it.
+  
+* A MySQL database that we aim to copy our data from. We will refer to it as ``source-db``.
+  
+* A target MySQL database that we aim to dump our ``source-db`` data. We will refer to it as ``target-db``.
+
+.. tip::
+
+    For the restore process, we recommend you pick a plan size that is large enough to store your data, so you can limit the downtime if you're performing a migration.
+
+The below example uses Aiven for MySQL databases for both, ``source-db`` and ``target-db``. You can create the databases by going to your running service for **Aiven for MySQL** > **Databases** > **Create a new database**.
+
+
+Backup the data
+---------------
+
+Variables
+'''''''''
+
+To backup the ``source-db`` data to a file called ``mydb_backup.sql`` you need to collect some information about your Aiven for MySQL ``source-db`` database. Go to your Aiven ``source-db`` service, from **Overview** page, find the following information:
+
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
+
+   * - Variable
+     - Description
+   * - ``SOURCE_DB_HOST``
+     - **Host** name for the connection
```

Comment:
Should we make these the same as they are shown in the Console? That is: Host, User, Port, Password, Database Name